cin_validator/rules/cin2022_23/rule_8839.py

```diff
@@ -46,7 +47,7 @@ def validate(
     # The rule originally asks for true, not 1, but an analyst coded it for 1, so their LA may use 1 and 0 instead, as such, it now check for either.
     df_check = df_check[
         df_check[ICPCnotRequired].isna()
-        | (~df_check[ICPCnotRequired].isin([1, "true"]))
+        | (~df_check[ICPCnotRequired].isin([1, "true", "1"]))
```

Comment:
This does the trick.
I'll change the double check to use .astype(str) so that we don't have to manually type out multiple versions of the same element.
`(~df_check[ICPCnotRequired].astype(str).isin([ "true", "1"])`